src/About.js

```diff
@@ -5,7 +5,7 @@ import './About.css';
 class About extends Component{
```

Comment:
question: Are you familiar with Function Components or just Class Components? In modern React versions (since 2019) we mostly use Functions and Classes got deprecated. Where did you learn about it?  When you look for tutorials/courses about React make sure they are at least from 2020.

Reply:
The first tutorial I had referred was based on class components. So when I started building this website, I was more familiar with class components. But now I have learnt Function Components and have been using function components in other recent projects.

Reply:
Great to know, perfect then :)